docs/data/galexie/admin_guide/prerequisites.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Prerequisites
+sidebar_position: 10
+---
+
+# Prerequisites
+
+### 1. Google Cloud Platform (GCP) Account
+
+Galexie exports Stellar ledger metadata to Google Cloud Storage (GCS), so you need a GCP account with:
+
+- Permissions to create a new GCS bucket, or
+- Access to an existing bucket with read/write permissions.
+
+### 2. Docker (Recommended)
+
+> **_NOTE:_** While it is possible to natively install Galexie (without Docker), this requires manual dependency management and is recommended only for advanced users.]
+
+Galexie is available as a Docker image, which simplifies installation and setup. Ensure you have Docker Engine installed on your system ([Docker installation guide](https://docs.docker.com/engine/install/)).
+
+## Hardware Requirements
```

Comment:
@mollykarcher I've added `hardware requirements` and `full history export` sections since your review, if you could take a look at them. Thanks!

Reply:
lgtm!